app/assets/javascripts/gallery.js.erb

```diff
@@ -655,6 +655,7 @@ $(document).ready(function(){
   $('.delEnvironment').on('click', function(){
     var name = encodeURIComponent($(this).attr('data-environmentName'))
     $('#confirmationModal').modal('show');
+    makeScreenreaderAlert('modalAlert', 'Dialog has appeared.');
```

Comment:
With the on(bs.dialog.show) action wouldn't all of these be duplicates/redundant?

Reply:
I thought so and it does work for the vast majority but it wasn't working for these particular ones because of the `return false` at the end of each of those functions which I did not touch out of fear those falses were needed for other reasons... to prevent other functions and processes from being called or modifying the page further. Would you like me to dive into each of these to see if we can remove that `return false`?

Reply:
probably just need an e.preventDefault(); rather than return false; so see if that resolves the issue.

Reply:
Will do